devTools/phpstan-src.neon

```diff
@@ -1,9 +1,9 @@
 parameters:
     ignoreErrors:
-        - '#Access to an undefined property PhpParser\\Node(.*?)#'
-        - '#Call to an undefined method PhpParser\\Node(.*?)#'
         - '#expects .*, array<string>\|bool\|string\|null given#'
         - '#cast array<string>\|bool\|string\|null to#'
-        - '#Offset array<string>\|bool\|string\|null does not#'
-        - '#Possibly invalid array key type array<string>\|bool\|string\|null#'
-        - '#Access to undefined constant PhpParser\\NodeTraverser::DONT_TRAVERSE_CURRENT_AND_CHILDREN#'
+        - '#Return type \(void\) of method Infection\\Visitor\\.*::.*\(\) should be compatible with return type .* of method PhpParser\\Node.*::.*\(\)#'
```

Comment:
I guess this is because we have `void` in our visitors, e.g. in `\Infection\Visitor\MutationsCollectorVisitor`

```php
public function leaveNode(Node $node): void
```

while it should be `null|int|Node|Node[]`

Is it better to fix the issue rather than ignore it?

Reply:
I have fixed this issue, by adding the correct return types, and returning `null` explicitly from some methods.